lib/matplotlib/finance.py

```diff
@@ -1,1390 +1,16 @@
 """
-A collection of functions for collecting, analyzing and plotting
-financial data.   User contributions welcome!
-
-This module is deprecated in 1.4 and will be moved to `mpl_toolkits`
-or it's own project in the future.
-
+Shim to provide backword compatibility for finance
 """
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
-
-import six
-from six.moves import xrange, zip
-
-import contextlib
-import os
 import warnings
-from six.moves.urllib.request import urlopen
-
-import datetime
-
-import numpy as np
-
-from matplotlib import colors as mcolors, verbose, get_cachedir
-from matplotlib.dates import date2num
-from matplotlib.cbook import iterable, mkdirs
-from matplotlib.collections import LineCollection, PolyCollection
-from matplotlib.lines import Line2D, TICKLEFT, TICKRIGHT
-from matplotlib.patches import Rectangle
-from matplotlib.transforms import Affine2D
-
-
-if six.PY3:
-    import hashlib
-
-    def md5(x):
-        return hashlib.md5(x.encode())
+from matplotlib.cbook import mplDeprecation
+try:
+    from mpl_finance import *
```

Comment:
Is `mpl_finance` an optional dependency?  Will people relying on the finance module will start getting this error immediately they install the new matplotlib?